pyspark/dl/models/textclassifier/README.md

```diff
@@ -17,15 +17,16 @@ or [20 Newsgroup dataset](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/theo-20/w
 ```{r, engine='sh'}
 $ [/tmp/news20]$ tree . -L 1
   .
-  ├── 20_newsgroup
-  └── glove.6B
+  ├── 20news-19997.tar.gz
+  └── glove.6B.zip
 ```
 
 then running the flowing script would automatically download the data during the first run.
 
 bigdl.sh would setup the essential environment for you and it would accept a spark-submit command as an input parameter.
 
 ```{r, engine='sh'}
+        PYTHONHASHSEED=...
```

Comment:
@122689305 , This has been fixed in Spark 2.2. we might need to remove this later. 
```
self.environment["PYTHONHASHSEED"] = os.environ.get("PYTHONHASHSEED", "0")
```